pallets/extended-assets/src/tests.rs

```diff
@@ -30,58 +30,14 @@
 
 #![cfg(feature = "wip")] // DEFI-R
 
-use crate::mock::*;
+use crate::mock::{Timestamp, *};
```

Comment:
* should import all, shouldn't?

Reply:
It should, but need to be explicitly importing Timestamp to differ the mock type from the one declared in lib.rs.